debian/pam-configs/authd.in

```diff
@@ -1,6 +1,6 @@
 Name: Authd authentication
 Default: yes
-Priority: 280
+Priority: 1050
```

Comment:
One thing, can't this be per-kind?

Since I'm a bit afraid of changing the priority also for other things that aren't `password`

Reply:
I'm not sure, tbh, but I wouldn't worry about it. If I understood correctly, it first considers the type before applying priorities.

Reply:
Ok, definitely authd wants to have lots of priority, so I can't see a problem here, we'll reiterate in case we have issues with this.